File: 01_Basico/02 - Controle_Selecao/Resposta_em_JavaScript/questao05.js
// Em javaScript: No lugar do if utilizamos a função sort, e por curiosidade usamos a função slice para clonar a lista de numeros, nesse caso não queria alterar o valor original de numeros)
const numero1 = parseInt(prompt("Digite um número: "))
const numero2 = parseInt(prompt("Digite um número: "))
const numero3 = parseInt(prompt("Digite um número: "))
const numeros = [numero1, numero2, numero3]
const clonado = numeros.slice().sort((a, b) => b - a)
console.log(`${clonado[0]},${clonado[1]},${clonado[2]}`)
console.log(numeros)

File: 01_Basico/02 - Controle_Selecao/Respsotas_em_Python/questao05.py
# Em python: No lugar do if utilizamos a função sorted, nesse caso apenas criamos uma lista organizada por sorted de forma reversa.
numero1 = int(input("Digite um número: "))
numero2 = int(input("Digite um número: "))
numero3 = int(input("Digite um número: "))
numeros_ordenados = sorted([numero1, numero2, numero3], reverse=True)
print(f"{numeros_ordenados[0]=},{numeros_ordenados[1]=},{numeros_ordenados[2]=}")